docs/src/running-validator/validator-reqs.md

```diff
@@ -42,7 +38,7 @@ your own internal usage. This use case includes exchanges and services built on
 Solana.
 
 In fact, the offical mainnet-beta API nodes are currently (Oct. 2020) run on GCE
-`n1-standard-32` (32 vCPUs, 120 GB memory) instances with 2048 GB SSD for
+`n2-standard-32` (32 vCPUs, 128 GB memory) instances with 2048 GB SSD for
```

Comment:
good eyes. :)

Reply:
And yet, I missed the date in the previous line :upside_down_face: